config/build.yml

```diff
@@ -203,11 +178,46 @@ command-hooks:
     command: null
 
 tests:
+  reports:
+    localDir: ${repo.root}/reports
+    remoteDir: reports
   run-server: false
   # This is used for ad-hoc creation of a server via `drush runserver`.
   server:
     port: 8888
     url: http://127.0.0.1:${tests.server.port}
+  # The URL of selenium server. Must correspond with setting in behat's yaml config.
+  selenium:
+    port: 4444
+    url: http://127.0.0.1:${tests.selenium.port}/wd/hub
+  chrome:
+    port: 9222
+    args: ""
+  chromedriver:
+    port: 9515
+    args: null
+  drupal:
+    # May be phpunit or run-tests.
+    test-runner: run-tests
+    sudo_run_tests: true
```

Comment:
Also looks like we have some inconsistency between usage of dashes and underscores. Sadly BLT does a lot of that but I'd prefer to try to standardize. Is there a reason not to use dashes everywhere?

Reply:
Good catch. Converted all property names to use `-`s